app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -1,6 +1,12 @@
-attributes :content_view_id, :content_view_name
-attributes :lifecycle_environment_id, :lifecycle_environment_name
-attributes :uuid
+child :content_view => :content_view do
+  attributes :id, :name
+end
+
+child :lifecycle_environment => :lifecycle_environment do
+  attributes :id, :name
+end
```

Comment:
@ehelms yeah, but who's contract are we honoring here, foreman's or katello's?  That's why I think we need both for now until we arrive at a consensus.